.github/workflows/integration-test-dumpling.yml

```diff
@@ -4,55 +4,55 @@ on:
     inputs:
       debug:
         type: boolean
-        description: 'Run the build with tmate debugging enabled'
+        description: "Run the build with tmate debugging enabled"
         required: false
         default: false
   push:
     branches:
       - master
       - release-*
     paths:
-      - 'dumpling/**'
-      - 'br/pkg/storage/**'
-      - 'br/pkg/utils/**'
-      - 'br/pkg/summary/**'
-      - 'store/helper/**'
-      - 'tablecodec/**'
-      - 'util/codec/**'
-      - 'parser/model/**'
+      - "dumpling/**"
+      - "br/pkg/storage/**"
+      - "br/pkg/utils/**"
+      - "br/pkg/summary/**"
+      - "store/helper/**"
+      - "tablecodec/**"
+      - "util/codec/**"
+      - "parser/model/**"
   pull_request:
     branches:
       - master
       - release-*
     paths:
-      - 'dumpling/**'
-      - 'br/pkg/storage/**'
-      - 'br/pkg/utils/**'
-      - 'br/pkg/summary/**'
-      - 'store/helper/**'
-      - 'tablecodec/**'
-      - 'util/codec/**'
-      - 'parser/model/**'
+      - "dumpling/**"
+      - "br/pkg/storage/**"
+      - "br/pkg/utils/**"
+      - "br/pkg/summary/**"
+      - "store/helper/**"
+      - "tablecodec/**"
+      - "util/codec/**"
+      - "parser/model/**"
 
 concurrency:
-  group: ${{ github.ref }}-${{ github.workflow }}
+  group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   mysql-5735:
-    uses: ./.github/workflows/integration-test-dumpling-common.yml
+    uses: ./.github/integration-test-dumpling-base.yml
     with:
       debug: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug }}
       mysql_version: 5.7.35
 
   mysql-8022:
-    uses: ./.github/workflows/integration-test-dumpling-common.yml
+    uses: ./.github/integration-test-dumpling-base.yml
```

Comment:
it's better to move to `.github/workflow-templates/integration-test-dumpling-base.yaml` ?

Reply:
Sounds good to me. But it looks like the new GHA workflow needs correct permission. I wonder if there is a possibility to make it automatic by tichi.